moto/ec2/models/elastic_ip_addresses.py

```diff
@@ -113,7 +112,7 @@ def __init__(self):
 
     def allocate_address(self, domain, address=None, tags=None):
         if domain not in ["standard", "vpc"]:
-            raise InvalidDomainError(domain)
+            domain = "vpc"
```

Comment:
The default value `standard` was only used in EC2-Classic. EC2-Classic was retired in August 2022.